etcdmgmt/etcd-mgmt.go

```diff
@@ -28,6 +64,19 @@ func StartEtcd() error {
 
 	initialAdvPeerUrls := "http://" + hostname + ":2380"
 
+	// Checking health of etcd cluster. If etcd daemon already running then
```

Comment:
@atinmu  By checking health of the cluster it seems to be problem here. We need to come up with different way like storing pidof etcd it self in etcd. @kshlm  you can also give your opinion in this case.